src/signature_pad.ts

```diff
@@ -87,6 +87,8 @@ export default class SignaturePad extends SignatureEventTarget {
 
     this.clear();
 
+    this.canvas.style.userSelect = 'none';
```

Comment:
This should be in the `on` function with the other style adjustments. Also remove this style in the `off` function.